src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Expected/GenerateDesignerFileExpected.cs

```diff
@@ -361,9 +361,9 @@ private String()
 		public partial class Styleable
 		{
 			
-			// aapt resource value: { 0x10100D2,0x7F040000,0x7F040000,0x7F040001 }
+			// aapt resource value: { 0x0,0x7F040000,0x7F040000,0x7F040001 }
 			public static int[] CustomFonts = new int[] {
-					16842962,
+					0,
```

Comment:
We don't want 0's here. This is a symtom of a problem where the android.jar file is not found. These id's should be resolved to the ones in the android.jar.

Reply:
So, I think the test is passing on main and failing on release/8.0.1xx only.

I think it might be this change we should just backport: https://github.com/xamarin/xamarin-android/commit/b1b6a762e3f8b511d1b257420180510462eae7c4

I will close this one.

Reply:
That second oen seems like the right one. We probably need both to be honest.